### DISCUSSION TITLE
Not Getting any dashboard after the installation of elastic-agent

### DISCUSSION THREAD

**Discussion by sakshamgarg07** [sherifabdlnaby/elastdocker#107] [Questions & Troubleshooting]
when I install the elastic-agent on the client computer it is not creating any dashboard. In standalone nothing is changing but in fleet server it is showing the client connected in healthy state but not receiving any kind of logs or data from the client please help.

**Comment by mostafahussein** [comment]
Hello @LAZYVAMPIRE ,
Can you please elaborate more on the steps you followed so we can reproduce the issue?
Thank you

**Comment by sakshamgarg07** [comment]
@mostafahussein whenever I connect elastic-agent it stucks on confim data as show
![elas](https://user-images.githubusercontent.com/114327885/206401662-fe998b6e-a8e3-4ea4-82e6-4beedb838a1d.jpg)
![elas](https://user-images.githubusercontent.com/114327885/206401662-fe998b6e-a8e3-4ea4-82e6-4beedb838a1d.jpg)
